### DISCUSSION TITLE
記事投稿時のリンクでのQiita Discussionsへのリンクの追加

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#150] [要望・アイデア・不具合]
以下の記事投稿時の意見や要望などのリンクにこちらのQiita Discussionsのリポジトリのリンクがあっても良いかもしれません！（正式運用となっているので）

![image](https://user-images.githubusercontent.com/25178045/155099476-43918ba8-dd88-4ca4-8cd1-af03d7d7f9d9.png)

**Comment by kyntk** [comment]
@simon-ritchie ありがとうございます！

本当ですね、ここにリンクがあるのに気づかず漏れていました 😅 

ありがとうございます！
Qiita Discussionsのリンクに差し替えます！

**Comment by kyntk** [comment]
@simon-ritchie 大変お待たせいたしました！
こちら対応してQiita Discussionsのリンクに差し替えました！

**Reply by simon-ritchie** [reply]
@kyntk さん

おお・・・ご対応ありがとうございます・・・！🙇‍♂️🎉
